contrib/docker/frontend-with-nginx/docker/run.sh

```diff
@@ -10,14 +10,27 @@ envsubst '$PORT' < /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/d
 
 # Inject runtime config into the client
 function inject_config() {
+  # Read runtime config from yaml file (it could be the backend one)
```

Comment:
Absolutely could not be the same as the backend one 😅 , it won't be filtered for secrets, so any secrets in the app config would be sent straight on to the frontend.